<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>653</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="open_Item"/>
    <addaction name="file_delete"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>窗口</string>
    </property>
    <addaction name="ToolBar_open"/>
    <addaction name="fileList_open"/>
    <addaction name="fileWindow_open"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="open_Item"/>
   <addaction name="save_item"/>
   <addaction name="save_file"/>
   <addaction name="file_delete"/>
   <addaction name="actionTest"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="filedockWidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>111</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>524287</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="filetree">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="ptptext">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="tabdockWidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>46</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>524287</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTabWidget" name="filetab">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="open_Item">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/icon/itemopen.jpg</normaloff>:/new/prefix1/icon/itemopen.jpg</iconset>
   </property>
   <property name="text">
    <string>打开项目</string>
   </property>
  </action>
  <action name="ToolBar_open">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>工具栏</string>
   </property>
  </action>
  <action name="fileList_open">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>文件列表</string>
   </property>
  </action>
  <action name="fileWindow_open">
   <property name="text">
    <string>文件窗口</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="file_delete">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/icon/delete.jpg</normaloff>:/new/prefix1/icon/delete.jpg</iconset>
   </property>
   <property name="text">
    <string>删除文件</string>
   </property>
  </action>
  <action name="save_item">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/icon/save.jpg</normaloff>:/new/prefix1/icon/save.jpg</iconset>
   </property>
   <property name="text">
    <string>保存项目</string>
   </property>
  </action>
  <action name="save_file">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/icon/timg.jpg</normaloff>:/new/prefix1/icon/timg.jpg</iconset>
   </property>
   <property name="text">
    <string>保存文件</string>
   </property>
  </action>
  <action name="new_file">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/icon/file.jpg</normaloff>:/new/prefix1/icon/file.jpg</iconset>
   </property>
   <property name="text">
    <string>新建文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
